server/src/main/java/org/elasticsearch/reservedstate/service/FileSettingsService.java

```diff
@@ -103,7 +101,7 @@ public Path operatorSettingsDir() {
     }
 
     public Path operatorSettingsFile() {
-        return operatorSettingsDir().resolve(SETTINGS_FILE_NAME);
```

Comment:
By not using the method we restrict our testing capability for unit tests.

Reply:
It's not super important, since we don't have a test written yet that say tries to move the directory somewhere else, but with this change that can never be tested.

Reply:
I'm also not a fan that we have a public method operatorSettingsDir() that tests rely on, but the internal code uses something else. What if we ever decided to change what the internals of that method do?

Reply:
OK, I'll use the method consistently in the class